lib/helpers.js

```diff
@@ -32,8 +32,8 @@ async function getDirectories (rootPath) {
 async function getAndroidPlatformAndPath () {
   const androidHome = process.env.ANDROID_HOME;
   if (!_.isString(androidHome)) {
-    log.error("ANDROID_HOME was not exported!");
-    return null;
+    log.error("ANDROID_HOME environment variable was not exported!");
+    return {platform: null, platformPath: null};
```

Comment:
Also, we can add better verification and check if the path set in ANDROID_HOME really exists on the filesystem